README.md

```diff
@@ -64,7 +67,7 @@ npm i -g azure-functions-core-tools
 To install v2:
 
 ```bash
-npm i -g azure-functions-core-tools@core --unsafe-perm true
+npm i -g azure-functions-core-tools@core
```

Comment:
`--unsafe-perm true` is only necessary when running as sudo and we already have the note a few lines above that says:
 > **NOTE**: npm can be used on all platforms. On unix platforms, you may need to specify `--unsafe-perm` if you are running npm with sudo. That's due to npm behavior of post install script.